textile/features.textile

```diff
@@ -442,7 +469,7 @@ h3(#realtimeclient). RealtimeClient
 h3(#realtime-connection). Connection
 
 * @(RTN1)@ @Connection@ connects to the Ably service using a "websocket":https://ably.com/topic/websockets connection. The "ably-js library":https://github.com/ably/ably-js supports additional transports such as Comet and XHR streaming; however non-browser client libraries typically use only a websocket transport
-* @(RTN2)@ The default host used for realtime "websocket":https://ably.com/topic/websockets connections is @realtime.ably.io@, and the following query string params should be used when opening a new connection:
+* @(RTN2)@ The default host used for realtime "websocket":https://ably.com/topic/websockets connections is @main.realtime.ably.net@, and the following query string params should be used when opening a new connection:
```

Comment:
again shouldn't this reference the `REC1` `Primary Domain`?

Reply:
Fixed https://github.com/ably/specification/pull/213/commits/27a16960b674d44ba91873e22f3203756e7d7900